.github/workflows/test-report.yaml

```diff
@@ -0,0 +1,47 @@
+name: Generate Longitudinal Test Report
```

Comment:
Nit: I think this shorter version conveys the same meaning, while fitting better into the left-hand side of the GitHub actions UI

```suggestion
name: Test Report
```

Reply:
Sure, makes sense to me